CMakeLists.txt

```diff
@@ -210,7 +209,7 @@ if(NOT DEFINED SNMALLOC_ONLY_HEADER_LIBRARY)
     add_shim(snmallocshim-1mib SHARED ${SHARED_FILES})
     target_compile_definitions(snmallocshim-1mib PRIVATE IS_ADDRESS_SPACE_CONSTRAINED)
   endif()
-
+  
```

Comment:
The only changes in this file are now whitespace.  Can you remove them to reduce churn?

Reply:
I have just rebased away the changes on `CMakeLists.txt`